app/lib/helpers.js

```diff
@@ -0,0 +1,14 @@
+const padTime = number => (number < 10 ? '0' + number : number);
```

Comment:
I needed some place for helpers.  Also I wanted to put code that is specific for library and re-usable to separate place